package.json

```diff
@@ -86,6 +86,7 @@
     "coverage:upload": "nyc report --reporter=text-lcov | coveralls",
     "pretest": "standard && standard-markdown",
     "test": "nyc mocha test/mocha-node-setup.js 'test/**/*-test.js'",
+    "posttest": "npm run-script build:ts && tsc test/typescript-validate.ts ",
```

Comment:
we have `validate:ts` which we don’t run with each `npm test`. Maybe better to move do 

```
 +    "postvalidate:ts": "tsc test/typescript-validate.ts", 
```

Instead?

Also tiny nitpick: trailing space at `.ts "`